tumblr-photo-video-ripper.py

```diff
@@ -107,6 +107,7 @@ def _download(self, medium_type, medium_url, target_folder):
                                         medium_name])
 
             medium_name += ".mp4"
+            medium_url = 'https://vt.tumblr.com/' + medium_name
```

Comment:
How comes this url?

Reply:
From web version

Reply:
@TainakaDrums I'm not sure for this change. Can anybody help verify this?

Reply:
I can confirm it that the previous medium_url is forwarded to this new medium_url in browser